src/redux/configureStore.js

```diff
@@ -0,0 +1,5 @@
+// import { combineReducers, createStore } from 'redux';
+
+// const store = createStore();
+
+// export default store;
```

Comment:
- [ ] Please kindly remove commented code code from your code base as it makes your code dirty and less readable.

Reply:
done